_data/rupav/rupav.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Rupav Jain
+institution: Netaji Subhas Institute of Technology
+profile_pic: rupav.jpg
+quote: "Control is just an illusion. Servers will crash no matter what you do!"
```

Comment:
Please avoid using quotes. From README:
> quote: YOUR-SENIOR-QUOTE # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.

Reply:
@ayan-b I have made the changes. Thanks for the review.